tensorflow/contrib/stateless/python/kernel_tests/stateless_random_ops_test.py

```diff
@@ -79,6 +80,21 @@ def testDeterminism(self):
             for s1, v1 in values:
               self.assertEqual(s0 == s1, np.all(v0 == v1))
 
+  def testShapeType(self):
+    with self.test_session(use_gpu=True):
+      for shape_dtype in [dtypes.int64]:
```

Comment:
Does it make sense to test both int64 and int32?

Reply:
Sorry I missed that one. Thanks for the update in the PR and help!